packages/client/lib/__tests__/OpenID4VCIClient.spec.ts

```diff
@@ -10,8 +10,8 @@ describe('OpenID4VCIClient should', () => {
 
   beforeEach(async () => {
     nock(MOCK_URL).get(/.*/).reply(200, {});
-    client = await OpenID4VCIClient.initiateFromURI({
-      issuanceInitiationURI: 'openid-initiate-issuance://?issuer=https://server.example.com&credential_type=TestCredential',
+    client = await OpenID4VCIClient.fromURI({
+      credentialOfferURI: 'openid-initiate-issuance://?issuer=https://server.example.com&credential_type=TestCredential',
```

Comment:
2 things:

1) The param name doesn't match the actual URI. This of course has to do with the rename in V11. Credential Offer is not a concept in V9, which then could confuse people.

2) The param should simply be renamed to uri for both versions, similar like the lower level class working with URIs. Previously we didn't know the actual concept of initiation would be completely renamed in the future.  The difference should become apparent from the actual value (openid-initiate-issuance vs openid-credential-offer). Since we now have 2 different URI scheme's and in the future you should also be able to have custom scheme's, simply renaming it to uri makes sense (similar to the method name)